Dockerfile

```diff
@@ -18,15 +18,20 @@ RUN apk add --update --no-cache \
       git \
       tzdata
 
-# RUN gem install bundler -v 2.3.11
-
 ENV APP_HOME /hdm
+ENV RAILS_ENV production
+ENV RAILS_SERVE_STATIC_FILES true
+ENV RAILS_LOG_TO_STDOUT true
+
 WORKDIR $APP_HOME
 
 COPY . $APP_HOME
 COPY config/hdm.yml.template $APP_HOME/config/hdm.yml
 
-RUN bundle check || (bundle config set --local without 'test' && bundle install)
+RUN bundle check || (bundle config set --local without 'development test linter' && bundle install)
+# RUN bundle exec rake assets:precompile # does not work on alpine
```

Comment:
dynamically on demand. as it is at the moment. precompiling like with the rake task does not work because dart breaks every time. something to do with glibc and musl, which i dont understand. but every search leads to this ticket: https://github.com/dart-lang/sdk/issues/40906

Reply:
Any reason why Alpine? Would it be possible with any other OS or Ruby container?

Reply:
Always compiling the assets on-the-fly can be OK(ish), but it can decrease (perceived) performance significantly. If there is any way to precompile the assets I would strongly suggest we do it.

Reply:
alpine is the smallest images available. most projects choose it. we can switch to ruby-thin. this is based on debian if i'm not wrong. its bigger than alpine, but we shouldn't run in this problem with the binaries 🤔